dotcom-rendering/src/web/components/Slideshow.tsx

```diff
@@ -0,0 +1,210 @@
+import { css, keyframes } from '@emotion/react';
+import { from, neutral, textSans, until } from '@guardian/source-foundations';
+import type { DCRSlideshowImage } from '../../types/front';
+import { CardPicture } from './CardPicture';
+
+/**
+ * It might look like you can change these and expect the animation to adapt accordingly
+ * but it probably won't 😞
+ */
+const HANG_TIME = 5;
+const FADE_TIME = 1;
+
+/**
+ * We pass the result of the Emotion `keyframes` utility (decided here) to the animation-name
+ * css property
+ */
+function decideAnimationName({ slideshowLength }: { slideshowLength: number }) {
+	const totalLoopTime = slideshowLength * (HANG_TIME + FADE_TIME);
+	// Calculate the percentages for how long each animation stage lasts
+	const stageOne = `0%`;
+	const stageTwo = `${(100 / totalLoopTime) * FADE_TIME}%`;
+	const stageThree = `${(100 / totalLoopTime) * (FADE_TIME + HANG_TIME)}%`;
+	const stageFour = `${
+		(100 / totalLoopTime) * (FADE_TIME + HANG_TIME + FADE_TIME)
+	}%`;
+	// Generate and return a keyframes name for this animation
+	return keyframes`
+		${stageOne} {opacity: 0;}
+		${stageTwo} {opacity: 1;}
+		${stageThree} {opacity: 1;}
+		${stageFour} {opacity: 0;}
+	`;
+}
+
+/**
+ * How long the whole animation loop will last for before it starts again
+ */
+function decideDuration({ slideshowLength }: { slideshowLength: number }) {
+	return slideshowLength * (HANG_TIME + FADE_TIME);
+}
+
+/**
+ *
+ * Decide how long each image should wait to start their animation sequence based
+ * on how many images there are and how long we show each image for.
+ */
+function decideDelay({
+	imageIndex,
+	slideshowLength,
+}: {
+	imageIndex: number;
+	slideshowLength: number;
+}) {
+	if (imageIndex === 0) return 0;
+	const totalLoopTime = slideshowLength * (HANG_TIME + FADE_TIME);
+	const delay = (totalLoopTime / slideshowLength) * imageIndex;
+	return delay;
+}
+
+const hideOnMobile = css`
+	${until.tablet} {
+		display: none;
+	}
+`;
+
+/**
+ * Applied to all images except the first one. The first image holds the space open for
+ * all the others to be positioned on top of
+ */
+const overlayImage = css`
+	position: absolute;
+	width: 100%;
+	top: 0;
+	left: 0;
+`;
+
+const animationStyles = ({
+	imageIndex,
+	slideshowLength,
+	imageSize,
+}: {
+	imageIndex: number;
+	slideshowLength: number;
+	imageSize: ImageSizeType;
+}) => {
+	const delay = decideDelay({
+		imageIndex,
+		slideshowLength,
+	});
+
+	const duration = decideDuration({
+		slideshowLength,
+	});
+
+	const animationName = decideAnimationName({
+		slideshowLength,
+	});
+
+	const animationCss = css`
+		animation-delay: ${delay}s;
+		animation-direction: normal;
+		animation-duration: ${duration}s;
+		animation-iteration-count: infinite;
+		animation-name: ${animationName};
+		animation-timing-function: linear;
+		opacity: 0;
```

Comment:
I haven't worked through this in depth, but would it be possible to add `animation-play-state: paused;` on hover? (Either when the parent container is hovered, or sibling selector from the current slide?)